tgg/seqr_scripts/sex_check.py

```diff
@@ -0,0 +1,208 @@
+        sex_ht = run_hails_impute_sex(mt, build, outdir, mt_name, male_fstat_threshold, female_fstat_threshold, aaf_threshold)  
+        sex_ht = sex_ht.annotate(ambiguous_sex=hl.is_missing(sex_ht.is_female),
+                                 sex_aneuploidy=(sex_ht.is_female) & hl.is_defined(sex_ht.normalized_y_coverage) & (sex_ht.normalized_y_coverage > y_cov_threshold) |
+                                 (~sex_ht.is_female) & hl.is_defined(sex_ht.normalized_y_coverage) & (sex_ht.normalized_y_coverage < y_cov_threshold))
+
+        sex_expr = (hl.case()
+            .when(sex_ht.ambiguous_sex, "ambiguous_sex")
+            .when(sex_ht.sex_aneuploidy, "sex_aneuploidy")
+            .when(sex_ht.is_female, "female")
+            .default("male"))
+
+        sex_ht = sex_ht.annotate(sex=sex_expr)
+        sex_ht = sex_ht.select(sex_ht.is_female, sex_ht.f_stat, sex_ht.n_called, sex_ht.expected_homs, sex_ht.observed_homs, sex_ht.sex, sex_ht.chrY_mean_cov, sex_ht.chr20_mean_cov, sex_ht.normalized_y_coverage)
+        
+    else:
+        sex_ht = run_hails_impute_sex(mt, build, outdir, mt_name, male_fstat_threshold, female_fstat_threshold, aaf_threshold)
+        sex_ht = sex_ht.annotate(ambiguous_sex=hl.is_missing(sex_ht.is_female))
+        sex_expr = hl.if_else(sex_ht.ambiguous_sex, "ambiguous_sex", hl.if_else(sex_ht.is_female, "female", "male"))
+        sex_ht = sex_ht.annotate(sex=sex_expr)
+        sex_ht = sex_ht.select(sex_ht.is_female, sex_ht.f_stat, sex_ht.n_called, sex_ht.expected_homs, sex_ht.observed_homs, sex_ht.sex)
+
+    outfile = f"{outdir}/sex_{mt_name}.txt"
+    sex_ht.export(outfile)
+    return(sex_ht)
+
+def main(args):
+    call_sex(**vars(args))
+
+if __name__ == '__main__': 
+    parser = argparse.ArgumentParser(description='This script infers the sex of samples')
+    parser.add_argument('-i', '--callset', required=True, help='Path to Callset MatrixTable')
+    parser.add_argument('-u', '--use-y-cov', help='bool for whether or not to use chrY coverage', action='store_true')
+    parser.add_argument('-y', '--y-cov-threshold', help='chrY coverage threshold', default=0.1)
+    parser.add_argument('-m', '--male-fstat-threshold', help='male_fstat_threshold for hails impute_sex', default=0.75)
+    parser.add_argument('-f', '--female-fstat-threshold', help='female_fstat_threshold for hails impute_sex', default=0.50)
+    parser.add_argument('-a', '--aaf-threshold', help='aaf_threshold for hails impute_sex', default=0.05)
+    parser.add_argument('-c', '--call-rate-threshold', help='call_rate_threshold required to use chrY variant', default=0.25)
+    # NOTE: this is an integer here because get_chr_cov expects chromosomes to be specified using their numbers
+    parser.add_argument('-n', '--normalization-contig', help='contig to calculate normalized_y_coverage', default="20")
+    
+    args = parser.parse_args()
+    main(args)
```

Comment:
```suggestion
    main(args)
    
```
(GitHub expects an empty line at the end of scripts)